package.json

```diff
@@ -17,14 +17,14 @@
     "author": "",
     "homepage": "https://github.com/cbtnuggets/eslint-config-cbtnuggets",
     "dependencies": {
-        "eslint": "^5.8.0",
+        "eslint": "^7.24.0",
```

Comment:
this... might break stuff.  But if we don't use any of the eslint plugins then it might be okay

Reply:
We have to go to the 6.x version to meet the minimum security requirement... I figured if we have to go a full version , why not two?

Reply:
I'm all for patching vulnerability holes.  Consumers can be better at linting.  Win win.  Although that makes me wonder if maybe this is a minor version bump?  Consumers can freely upgrade, but should they feel free to downgrade?